app/uk/gov/hmrc/eoricommoncomponent/frontend/connector/EnrolmentStoreProxyConnector.scala

```diff
@@ -42,7 +42,7 @@ class EnrolmentStoreProxyConnector @Inject() (http: HttpClient, appConfig: AppCo
     groupId: String
   )(implicit hc: HeaderCarrier, reads: Reads[EnrolmentStoreProxyResponse]): Future[EnrolmentStoreProxyResponse] = {
     val url =
-      s"$baseUrl/$serviceContext/enrolment-store/groups/$groupId/enrolments?type=principal&service=HMRC-CUS-ORG"
+      s"$baseUrl/$serviceContext/enrolment-store/groups/$groupId/enrolments?type=principal&service=HMRC-CUS-ORG"  // TODO should this be [SERVICE]
```

Comment:
This probably require to pass the HMRC-SERVICE-ORG

Reply:
I'm wondering if we need to have service here. I'll take a look at the enrolment store proxy endpoint, what is required for it

Reply:
It looks like this need to be a service specific. Good catch!

Reply:
I'll fix this then.

Reply:
We can remove this service name from this call, because it's not required for principal enrolments